python-3-alpine/Docker/entrypoint.bash

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+
+if [[ -v DEBUG ]] && [[ "$DEBUG" == true ]]; then
+  set -x
+fi
+
+
+################################################################################
+# If user/group ID specified, ensure a user/group with correct ID is available
+################################################################################
+source /opt/configure_user_and_group.bash
+
+
+################################################################################
+# Define command to be called
+################################################################################
+COMMAND="python"
+
+################################################################################
+# Pass down additional arguments
+################################################################################
+for ARGUMENT in "${@}"; do
+  COMMAND+=" $ARGUMENT"
+done
+
+
+################################################################################
+# Perform mvn command, with correct user ID & group ID
```

Comment:
Copy/Paste typo. I think we should make it generic, to avoid those kind of errors:

```bash
# Perform command ...
```

Reply:
I think I just need to pay more attention next time, I fixed it :)